README.md

```diff
@@ -58,7 +58,7 @@ In [infrastructure](infrastructure/), we present all the services which should b
 
 ## Laboratory Operator
 
-The Laboratory Operator (LabOperator) implements the backend logic necessary to spawn new laboratories starting from a predefined template. The instruction to install and configure the LabOperator are detailed [here](operators).
+The Laboratory Operator (LabOperator) implements the backend logic necessary to spawn new laboratories and persistent VMs starting from a predefined template. The instruction to install and configure the LabOperator are detailed [here](operators).
```

Comment:
@giorio94 Hasn't be renamed that operator?
Btw, the name "lab operator" appears still here (I believe by mistake): https://github.com/netgroup-polito/CrownLabs/blob/cc8f2df7f96952970c255a30d7de0d9cee34c192/operators/pkg/instance-controller/controller_test.go#L26

Reply:
Yes, the correct name (at least from my understanding) should be Instance Operator. However, there are still many references both in the documentation and in the code to the old name, which should be corrected.